spring-cloud-gcp-autoconfigure/src/main/java/org/springframework/cloud/gcp/autoconfigure/datastore/GcpDatastoreAutoConfiguration.java

```diff
@@ -66,7 +60,7 @@
 
 	private final Credentials credentials;
 
-	private final String emulatorHost;
```

Comment:
This is something internal in this class.
If the `host` is really just about an emulator, then it would be better to keep the property as `emulatorHost`

Reply:
The host is not only about the local emulator.. Users could start an emulator in a host and connect to it using another application. This is a valid use even if not often. If we keep the `emulatorHost` property, we will have both `emulatorHost` and `emulator` properties, I really think we should keep some kind of pattern here. I will solve the other discussions and wait for some advice here.

Reply:
Check my review comment about this issue. Thank you.